### ISSUE TITLE
Can I live stream my web cam video to gemini vision api?

### ISSUE THREAD

**Issue by Munsyari** [google-gemini/generative-ai-python#372]

**Comment by MarkDaoust** [comment]
@vishwakneelamegam correct. The API supports streaming output, but no streaming input. This is impossible right now.

**Comment by vishwakneelamegam** [comment]
I have made a workaround with Open AI https://github.com/vishwakneelamegam/openai-video-narrator.

**Comment by MarkDaoust** [comment]
Update. Yes, you can totally do this. 

Start here: https://github.com/google-gemini/cookbook/blob/main/quickstarts/Get_started_LiveAPI.ipynb
